plugins/org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/model/GeneratedJavaFileAccess.xtend

```diff
@@ -25,9 +26,17 @@ class GeneratedJavaFileAccess extends JavaFileAccess {
 		if (!markedAsGenerated)
 			throw new IllegalArgumentException("It's always generated");
 	}
-
+	
+	def setTypeComment(StringConcatenationClient javaContent) {
```

Comment:
I think setters should return void in general. Also in JavaFileAccess. Is the return value of JavaFileAccess.content used somewhere?